src/Interpreters/InterpreterExplainQuery.cpp

```diff
@@ -72,7 +72,7 @@ namespace
             if (query_info.view_query)
             {
                 ASTPtr tmp;
-                StorageView::replaceWithSubquery(select, query_info.view_query->clone(), tmp);
+                StorageView::replaceWithSubquery(select, query_info.view_query->clone(), tmp, query_info.is_parameterized_view);
```

Comment:
Comment as above -- may be pass `query_info` as a whole? (just to make it more generic and avoid adding an argument `is_create_parametrized_view` in so many methods)